src/boot/is_debug.c

```diff
@@ -35,11 +41,15 @@ void rmonPrintf(const char* fmt, ...) {
     va_list args;
     va_start(args, fmt);
 
+#ifdef OOT_DEBUG
     _Printf(is_proutSyncPrintf, NULL, fmt, args);
+#endif
 
     va_end(args);
 }
 
+#ifdef OOT_DEBUG
+
 void* is_proutSyncPrintf(void* arg, const char* str, size_t count) {
```

Comment:
Why empty line
I mean why not my point is more "agree arbitrarily or standardize yes/no"

Reply:
I originally had two types of ifdefs:

* "Small" blocks that had no space between them, and had no /* OOT_DEBUG */ comment on the endif
* "Large" blocks that had spaces around them and had an /* OOT_DEBUG */ comment on the endif

But this distinction is gone now so I removed the extra lines